### DISCUSSION TITLE
is the project died?

### DISCUSSION THREAD

**Discussion by xfxpositions** [just-js/just#63] [General]
there is no commit since 2022, what happened?
is still going?

**Comment by kungfooman** [comment]
no code not life :zombie:

**Comment by billywhizz** [comment]
life got in the way i am afraid. i have another experiment i am working on that i might release soon. i might replace just-js with it, as i have learned a lot since i built this. if you have any suggestions/thoughts please let me know. hopefully i can spend more time in coming months.
